packages/help-center/src/components/help-center.tsx

```diff
@@ -102,9 +106,10 @@ const HelpCenter: React.FC< Container > = ( { handleClose, hidden } ) => {
 	const { hasActiveChats, isEligibleForChat } = useChatStatus( 'wpcom_messaging', false );
 	const { isMessagingScriptLoaded } = useZendeskMessaging(
 		'zendesk_support_chat_key',
-		isEligibleForChat || hasActiveChats,
+		( isHelpCenterShown && isEligibleForChat ) || hasActiveChats,
```

Comment:
Just an FYI that this change broke precancelation chat and other use cases where chat was directly triggered: https://github.com/Automattic/wp-calypso/pull/85501.

Reply:
Hi Igor :wave:, thanks for the ping, did you need help with the fix?

It sounds as though the precancelation chat is relying on some kind of global state set by `useZendeskMessaging` and assumes the `enabled` param would always be passed in as `true` elsewhere in the stack?

Separating the two conflated concerns like you are in https://github.com/Automattic/wp-calypso/pull/85501/files?w=1#diff-ac4b8e65c1a3ef7336ecca35fe4c1be3628a73a08e045b237ad373f2366d8d9bR126-R129 looks like a suitable fix to me.